ansible/roles/build-grsec-kernel/defaults/main.yml

```diff
@@ -45,3 +45,9 @@ linux_checksums_url: "{{ linux_base_url }}/sha256sums.asc"
 linux_source_directory: "{{ download_directory }}/linux-{{ linux_kernel_version }}"
 
 gpg_keyserver: hkps.pool.sks-keyservers.net
+
+# Assumes 64-bit (not reading machine architecture dynamically.)
+grsecurity_deb_package: linux-image-{{ linux_kernel_version }}-grsec_{{ linux_kernel_version }}-grsec-10.00.Custom_amd64.deb
+
+grsecurity_use_ccache: true
+ccache_directory: /usr/local/lib/ccache
```

Comment:
Actually, that var isn't even used—thanks for catching! Since `sudo` isn't required for building, `ccache` automatically uses the `$HOME/.ccache`, which is sane. 

Similarly, `linux_source_directory` should be located in user's home, but postponing that for now since the role doesn't actually enforce running as a non-root user.